### ISSUE TITLE
[1] - Responsive Navbar

### ISSUE THREAD

**Issue by Potato-29** [Potato-29/turf-booking-site#1]
The current navbar is only designed for desktops, you can work on it by adding responsiveness to the existing navbar

I'm looking for something basic like this
CLOSED STATE:
![image](https://github.com/user-attachments/assets/7582b7cd-f469-44ee-828e-205aac8ab029)

OPEN STATE:
![image](https://github.com/user-attachments/assets/ec5da154-f078-4023-9045-ee6caf29bb67)
NOTE: it should show the active route

Please read the CONTRIBUTING.md file before starting this task.

Thank you <3

**Comment by js-hari** [comment]
Hi! I would like to contribute to this issue. Please assign me to it, and I’ll get started right away.

**Comment by Potato-29** [comment]
Sure, thanks for the help @js-hari :)

**Comment by harsh0728** [comment]
Hii I would like to work on it and solve this issue Assign this to me.

**Comment by Potato-29** [comment]
Hi @harsh0728, as you can see, the issue has already been assigned to someone else. Please star the repo so you can stay in touch with any new issues you might want to work on

**Comment by Potato-29** [comment]
Hi @js-hari, any updates on this, If you are not interested in this then I can assign to someone else

**Comment by js-hari** [comment]
I’ll review and update accordingly. Let me know if there's anything specific you'd like me to address.

**Comment by Potato-29** [comment]
nothing specific, just wanted to know if you've started working on this?

**Comment by Potato-29** [comment]
Hi @harsh0728, I opened a new issue, you can check it out and see if you wanna contribute :)

**Comment by Potato-29** [comment]
Hi @js-hari, any updates?

**Comment by js-hari** [comment]
Heyy @js-hari , I'll create a PR by EOD.
